src/views/compute/wizard/NetworkConfiguration.vue

```diff
@@ -26,16 +26,40 @@
     :scroll="{ y: 225 }"
   >
     <template slot="ipAddress" slot-scope="text, record">
-      <a-input
-        style="width: 150px;"
-        :placeholder="$t('label.ipaddress')"
-        @change="($event) => updateNetworkData('ipAddress', record.id, $event.target.value)" />
+      <a-form-item>
+        <a-input
+          style="width: 150px;"
+          v-decorator="['ipAddress' + record.id, {
+            rules: [{
+              validator: validatorIpAddress,
+              cidr: record.cidr,
+              networkType: record.type
+            }]
+          }]"
```

Comment:
@utchoang Could you add a placeholder in the ipaddress input with the cidr ?
Also could you add the cidr in the expanded row render of the network ?

Reply:
@davidjumani `Could you add a placeholder in the ipaddress input with the cidr ?`
Do you mean like that?
![image](https://user-images.githubusercontent.com/13766648/92439064-4dde5a80-f1d4-11ea-8dfc-fcdf1f7a8e79.png)

Reply:
@utchoang Perfect! @rhtyd Comments ?